docs/framework/migration-guide/retargeting/new_ambiguous_dispatcherinvoke_overloads_could_result_in_different_behavior.md

```diff
@@ -0,0 +1,11 @@
+### New (ambiguous) Dispatcher.Invoke overloads could result in different behavior
+
+|   |   |
+|---|---|
+|Details|The .NET Framework 4.5 adds new overloads to Dispatcher.Invoke that include a parameter of type @System.Action. When existing code is recompiled, compilers may resolve calls to Dispatcher.Invoke methods that have a @System.Delegate parameter as calls to Dispatcher.Invoke methods with an @System.Action parameter. If a call to a Dispatcher.Invoke overload with a @System.Delegate parameter is resolved as a call to a Dispatcher.Invoke overload with an @System.Action parameter, the following differences in behavior may occur:<br /><ul><li>If an exception occurs, the &lt;xref:System.Windows.Threading.<xref:System.Windows.Threading.Dispatcher.UnhandledException>Filter&gt; and <xref:System.Windows.Threading.Dispatcher.UnhandledException> events are not raised. Instead, exceptions are handled by the <xref:System.Threading.Tasks.TaskScheduler.UnobservedTaskException?displayProperty=name> event.</li><li>Calls to some members, such as <xref:System.Windows.Threading.DispatcherOperation.Result>, block until the operation has completed.</li></ul>|
```

Comment:
Oh boy, ahaha. :( I was doing a bunch of replaces that day... and was too aggressive about it. Lol.